games-strategy/lgeneral/lgeneral-1.3.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A turn-based strategy engine"
+DESCRIPTION="LGeneral is a turn-based strategy engine heavily inspired by Panzer General. \
+You play single scenarios or whole campaigns turn by turn against a human \
+player or the AI. Entrenchment, rugged defense, defensive fire, surprise \
+contacts, surrender, unit supply, weather influence, reinforcements and other \
+implementations contribute to the tactical and strategic depth of the game."
+HOMEPAGE="http://lgames.sourceforge.net/"
+COPYRIGHT="2000, 2015 Michael Speck, Leo Savernik, Peter Ivanyi"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sf.net/lgeneral/lgeneral-$portVersion.tar.gz"
+CHECKSUM_SHA256="6e95a84d7772f0bd2d64edac8c2aa45c6f750ae47bfb9dc203fe53a089e5562c"
+SOURCE_URI_2="http://prdownloads.sf.net/lgeneral/pg-data.tar.gz"
```

Comment:
Sorry, I had not noticed that the recipe had a SOURCE_URI_2. If you want you can remove the **`pr`** in **`prdownloads.sf.net`** to obtain:
```
SOURCE_URI_2="http://downloads.sf.net/lgeneral/pg-data.tar.gz"
```
**`prdownloads.sourceforge.net`** was the name SF had chosen a long time ago. Then they chose to "switch" to **`downloads.sourceforge.net`** and, if I'm not mistaken, they even removed **`prdownloads.sourceforge.net`** for some time, before adding it back. But I guess SF prefers **`downloads.sourceforge.net`**.